distuv/beta.go

```diff
@@ -0,0 +1,99 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+	"math/rand"
+)
+
+// Beta implements the Beta distribution, a two-parameter continuous distribution
+// with support over the positive real numbers.
+//
+// The beta distribution has density function
+//  x^(α-1) * (1-x)^(β-1) * Γ(α+β) / (Γ(α)*Γ(β))
+//
+// For more information, see https://en.wikipedia.org/wiki/Beta_distribution
+type Beta struct {
+	// Alpha is the left shape parameter of the distribution. Alpha must be greater
+	// than 0.
+	Alpha float64
+	// Beta is the right shape parameter of the distribution. Beta must be greater
+	// than 0.
+	Beta float64
+
+	Source *rand.Rand
+}
+
+// ExKurtosis returns the excess kurtosis of the distribution.
+func (be Beta) ExKurtosis() float64 {
```

Comment:
Can we use `b` as the receiver throughout, and `alpha` and `beta` as the locals, please?